lib/tags/group-settings.tag

```diff
@@ -188,6 +190,7 @@
                     let successMessage = i18n.titleSuccessMessage;
                     self.refs.title.classList.remove('graphjs-error');
                     self.refs.title.classList.add('graphjs-success');
+                    self.refs.title.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.title refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above